### PR TITLE
Cleanup actions

### PR DESCRIPTION
**Pull request by relrelb**
Cleanup GitHub actions a bit.
Shouldn't affect their behavior.

### CODE REVIEW

File: .github/workflows/test_web.yml [hunk 1]
```diff
@@ -8,37 +8,33 @@ on:
...
-    name: Test node ${{ matrix.node_version }} / rust ${{ matrix.rust_version }} / ${{ matrix.os }}
+    name: Test Node.js ${{ matrix.node_version }} / Rust ${{ matrix.rust_version }} / ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     continue-on-error: ${{ matrix.rust_version == 'nightly' }}
     strategy:
       fail-fast: false
       matrix:
         # We most likely don't care about rust versions here, we'll catch those issues in "test rust"
-        node_version: ['10', '12']
+        node_version: ['12', '14']
```
Comment (madsmtm) [inline .github/workflows/test_web.yml:18]: Don't know much about Node packages, but is it significant that we're bumping the minimum supported Node version here?

Comment (relrelb) [reply .github/workflows/test_web.yml:18]: Thanks for the feedback.
Node.js 10 is over 2 years old by now, and Node.js 14 is the current LTS. For me it sounds okay.

Comment (madsmtm) [reply .github/workflows/test_web.yml:18]: Fair 👍

File: .github/workflows/release_nightlies.yml [hunk 2]
```diff
@@ -125,40 +121,39 @@ jobs:
...
 
-      - name: Package linux/mac
+      - name: Package macOS / Linux
         if: matrix.os == 'macOS-latest' || matrix.os == 'ubuntu-latest'
         run: |
-          cp target/release/ruffle_desktop release/ruffle
-          cd release
-          tar -czvf ../release.tar.gz *
+          cp target/release/ruffle_desktop package/ruffle
+          tar -czvf release.tar.gz package/*
```
Comment (Herschel) [inline .github/workflows/release_nightlies.yml:148]: I believe this adds the `package/` path into the zip file, which I'd like to avoid if possible.

**Review by Herschel — APPROVED** [review]

**Comment by Herschel** [comment]
Thank you!

**Comment by torokati44** [comment]
This might have caused the web build on GitHub Actions to not find `wasm-opt`, and since this is silently ignored (as per https://github.com/ruffle-rs/ruffle/pull/2482#issuecomment-760489481), the nightlies might be unoptimized now?
Just by looking at the build logs...
It might be because of this: https://github.com/ruffle-rs/ruffle/pull/2974/commits/10e567d07a13992f922f5fc59cf04b63aec6dc0d#diff-6828a02bc62b64072cf21e8e74ccdbef7a94667770656e015276d7e330c82254L114

Although that line really shouldn't be necessary?: https://github.com/marketplace/actions/setup-miniconda#use-a-default-shell

**Pull request merged**